util/querysets.py

```diff
@@ -0,0 +1,43 @@
+from projects.models import Project
+from allocations.models import Allocation
+from django.db.models import Max, Prefetch
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class CCQuerySets:
```

Comment:
This file feels like it's automagically doing something just based on its name. Is that true? Are these being loaded/exposed somehow? If so, can we document that this is the case at the top of the file.

Reply:
This is documented at: https://docs.djangoproject.com/en/1.11/topics/logging/#naming-loggers

> By convention, the logger name is usually __name__, the name of the python module that contains the logger. This allows you to filter and handle logging calls on a per-module basis. However, if you have some other way of organizing your logging messages, you can provide any dot-separated name to identify your logger